Sources/FioriARKit/ARCards/Views/Authoring/AnnotationSceneAuthoringView.swift

```diff
@@ -0,0 +1,148 @@
+//
+//  AnnotationSceneAuthoringView.swift
+//
+//
+//  Created by O'Brien, Patrick on 9/13/21.
+//
+
+import SwiftUI
+
+public struct AnnotationSceneAuthoringView: View {
```

Comment:
Proposal: Name change to `SceneAuthoringView`
Reason: he view allows to author a scene with its annotations and there is no `AnnotationScene`

Reply:
My reasoning for this was to be specific in case the project grows. Although I'm not sure what other Authoring flow could exist, so I agree. SceneAuthoringFlow works better.